guidelines/index.html

```diff
@@ -1198,7 +1198,7 @@ <h5>Adjust color</h5>
 					</section>
 					<section class="requirement" data-status="exploratory">
 						<h5>Adjust background</h5>
-						<p class="requirement-text">Patterns, designs or images placed behind text are avoided or can be removed by the user.</p>
+						<p class="requirement-text">Patterns, designs, or images placed behind text are avoided or can be removed by the user.</p>
```

Comment:
Another one where I think it should be an "and" not "or" construct. Again, not related to your change.